src/applications/virtual-agent/components/chatbox/Chatbox.jsx

```diff
@@ -15,7 +15,7 @@ import {
 } from './loadingStatus';
 import { storeUtterances, LOGGED_IN_FLOW, IN_AUTH_EXP } from './utils';
 
-const ONE_MINUTE_IN_MS = 60_000;
+// const ONE_MINUTE_IN_MS = 60_000;
```

Comment:
Please delete

Reply:
We'll make a point of removing this in the future.

Reply:
We'll make a point of removing this for sure.